src/olympia/addons/management/commands/process_addons.py

```diff
@@ -72,14 +72,20 @@ def get_recalc_needed_filters():
                 'task': recalculate_post_review_weight,
                 'queryset_filters': get_recalc_needed_filters(),
             },
-            'resign_addons_for_cose': {
-                'task': sign_addons,
+            'bump_and_resign_addons': {
+                'task': bump_and_resign_addons,
                 'queryset_filters': [
                     # Only resign public add-ons where the latest version has been
                     # created before the 5th of April
                     Q(
                         status=amo.STATUS_APPROVED,
                         _current_version__created__lt=datetime(2019, 4, 5),
+                        disabled_by_user=False,
+                        type__in=(
+                            amo.ADDON_EXTENSION,
+                            amo.ADDON_LPAPP,
```

Comment:
FIXME: We don't need langpacks

Reply:
Aren't they signed too?

Reply:
https://github.com/mozilla/addons-server/pull/22109/commits/c60c833ee3cd1b543ea8f7c2cc9c141c1d54fff4 - we realized during our meeting about the comms for this that we didn't need (and really shouldn't!) bump langpacks, they are each meant for a specific version of Firefox from years ago that will still accept SHA-1 signatures...